frontend/src/components/TheLeafletMap.vue

```diff
@@ -32,6 +33,7 @@ const chartStore = useChartsStore()
 const { mapObject } = storeToRefs(mapStore)
 const minReachSelectionZoom = 7
 const mapInitialZoom = 3
+const accessToken = "AAPK7e5916c7ccc04c6aa3a1d0f0d85f8c3brwA96qnn6jQdX3MT1dt_4x1VNVoN8ogd38G2LGBLLYaXk7cZ3YzE_lcY-evhoeGX"
```

Comment:
might want to make this en environment variable but I guess okay to hard-code it here if you're alright with it being publicly accessible in Github

Reply:
yes, good point we can keep it for AGU and I will invalidate and recreate a new one after AGU that should be an environment variable, this could be abused.